third_party/terraform/website/docs/r/compute_instance.html.markdown

```diff
@@ -337,7 +337,7 @@ The `node_affinities` block supports:
 * `operator` (Required) - The operator. Can be `IN` for node-affinities
     or `NOT_IN` for anti-affinities.
 
-* `value` (Required) - The values for the node affinity label.
+* `value` (Required) - The value for the node affinity label.
```

Comment:
Other way around! The actual field is `values`. You can see it in the nodeAffinities block here: https://cloud.google.com/compute/docs/reference/rest/v1/instances

Reply:
nice catch! resolved